services/prefect/flows/generated_flows.py

```diff
@@ -123,13 +123,11 @@ def create_flow_for_scraper(ix: int, cls: Type[DatasetBase], schedule=True):
             d = create_scraper(cls)
             fetched = fetch(d)
             normalized = normalize(d)
-            validated = validate(d)
             done = put(d, connstr)
 
             d.set_upstream(sentry_sdk_task)
             normalized.set_upstream(fetched)
-            validated.set_upstream(normalized)
-            done.set_upstream(validated)
```

Comment:
I feel like we should probably actually remove the `validate()`, `_validate()` and `DatasetBase.validate()` functions or else keep these lines but comment them out with a comment that we haven't fully implemented the validate logic and we're disabling it since it costs us money.

Reply:
Yeah, fair point--I've gone ahead and removed the validate functions, and references to these steps in the docs 🙏